configs/errors.eslintrc.json

```diff
@@ -130,7 +130,22 @@
         }
       }
     ],
-    "import/no-extraneous-dependencies": "error"
+    "import/no-extraneous-dependencies": "error",
+    "import/order": [
+      "error",
+      {
+        "alphabetize": {
+          "order": "asc"
+        }
+      }
+    ],
+    "sort-imports": [
+      "error",
+      {
+        "ignoreDeclarationSort": true,
+        "ignoreCase": true
+      }
+    ]
```

Comment:
```suggestion
    "import/no-extraneous-dependencies": "error",
    "import/order": [
      "error",
      {
        "alphabetize": {
          "order": "asc",
          "caseInsensitive": false
        }
      }
    ],
    "sort-imports": [
      "error",
      {
        "ignoreDeclarationSort": true,
        "ignoreCase": false
      }
    ]
```

Reply:
Hm... I had hoped that `"source.fixAll.eslint": "explicit",` would be enough to automatically trigger the resorting on save. Let me investigate a bit. I'm happy with whichever requires the least effort :-).

Reply:
@planger, did you find that with the existing code, you didn't get auto-formatting on save according to rules defined? Since they're ESLint rules and all auto-fixable, I believe the preference added by @haydar-metin 
```json
    "editor.codeActionsOnSave": {
      "source.fixAll.eslint": "explicit"
    }
```
should trigger reordering, and that's what I'm seeing locally, but it sounds like you didn't get that behavior?

Reply:
@colin-grant-work Interesting, I retried today and now everything seems to work very well. Maybe the ESLint extension was in a weird state when switching branches. Anyway, now organize imports is triggered on save and agrees with the ESLint rules. Sorry for raising the now non-issue earler.